### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by antoan2**
Creation of a footer with links to resources and to 2 pages:
- The about page where the team will be presented
- The methodology page where the different computation methodology will be presented

There is also a small refactor of the home page to use dash bootstrap components and the addition of a BR in the index layout.

### CODE REVIEW

File: app/src/apps/footer.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                dbc.Col(
+                    html.P("2020 copyright dataforgood", style={"text-align": "center", "margin": "20px"}),
+                    align="center",
+                )
+            ),
+        ],
+        style={"background": "lightgrey", "margin": "0px"},
+        fluid=True,
+    ),
+    # style={"position": "absolute", "bottom": "0"},
```
Comment (Tiyop) [inline app/src/apps/footer.py:68]: I think we can stick it to the bottom of the page with some custom CSS:
`style={"position": "fixed", "bottom": "0", "left": "0", "right": "0"},`

File: app/src/apps/methodology.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Et on a utilisé les facteurs d'émissions suivants :
+- Facteur 1
+- Facteur 2
+"""
+)
+
+layout = html.Div(
+    children=[intro]
+    + [html.A(id="methodologie_odrive"), methodology_odrive]
```
Comment (Tiyop) [inline app/src/apps/methodology.py:41]: Good idea !

**Review by Tiyop — APPROVED** [review]
That's great to have a footer + few extra pages (e.g. methodology)
Thanks

File: app/src/apps/footer.py (hunk 1 above)
Comment (antoan2) [reply app/src/apps/footer.py:68]: It is not enough :s The idea is to have the footer at the bottom of the page, and push it if there is other content in the page.
I will think about this today, and will see if I find solutions.
image

Comment (antoan2) [reply app/src/apps/footer.py:68]: We will work on this later, I am adding an issue :
https://github.com/dataforgoodfr/batch7_beges/issues/24

**Pull request merged**